app/pods/publications/publication/translations/documents/template.hbs

```diff
@@ -113,9 +113,16 @@
                           @animation="shift"
                           @placement="bottom">
                     <Auk::Dropdown>
-                      <Auk::Dropdown::Item @skin="primary" {{on "click" (fn this.openPieceEditModal piece)}}>
+                      <Auk::Dropdown::Item
+                        {{on "click" (fn this.openPieceEditModal piece)}}>
                         {{t "edit-translation-details"}}
                       </Auk::Dropdown::Item>
+                      {{!-- TODO: make @disabled work for dropdown item --}}
+                      <Auk::Dropdown::Item @skin="danger"
+                        @disabled={{compute this.isDeleteDisabled piece}}
```

Comment:
Heb je meer info over die `compute` helper in Ember, want die ken ik niet.

Reply:
Met de hulp van @madnificent gevonden dat dit een helper is uit ember-composable-helpers. Het is een vreemd patroon om een action te maken voor iets wat eigenlijk een getter is. Hier komen normaal betere helpers voor in Ember in de toekomst. Tot dan zou ik voorzichtig zijn met het gebruik ervan.